scrc/utils/test_utils.py

```diff
@@ -0,0 +1,208 @@
+
+ZH_Sozialversicherungsgericht_test_header = ['Sozialversicherungsgerichtdes Kantons ZürichIV.2014.00602', 'II. Kammer', 'Sozialversicherungsrichter Mosimann, Vorsitzender', 'Sozialversicherungsrichterin Käch', 'Sozialversicherungsrichterin Sager', 'Gerichtsschreiberin Kudelski', 'Urteil vom 11. August 2015', 'in Sachen', 'X._', 'Beschwerdeführerin', 'vertreten durch Rechtsanwalt Dr. Kreso Glavas', 'Advokatur Glavas AG', 'Markusstrasse 10, 8006 Zürich', 'gegen', 'Sozialversicherungsanstalt des Kantons Zürich, IV-Stelle', 'Röntgenstrasse 17, Postfach, 8087 Zürich', 'Beschwerdegegnerin', 'weitere Verfahrensbeteiligte:', 'Personalvorsorgestiftung der Y._', 'Beigeladene']
+
+namespace_de = {'language': Language.DE}
+namespace_fr = {'language': Language.FR}
+namespace_it = {'language': Language.IT}
+
+
+
+
+class TestCourtCompositionExtractingFunctions:
+    """
+    This class tests whether the court composition extracting functions give the correct procedural participation for a given input. If the output is incorrect, an error is shown. 
+    """
+
+    def court_composition_setup(court, test_list, namespace):
+        """
+        :param court:       the court to be tested
+        :param test_list:   the header to be tested
+        :param namespace:   the header's language
+        :return:            the CourtComposition
+        """
+        test_string = ' '.join(map(str, test_list))
+        sections = {}
+        sections[Section.HEADER] = test_string
+        return court(sections, namespace)
+        
+    zg_vg = court_composition_setup(c.ZG_Verwaltungsgericht, ZG_Verwaltungsgericht_test_header, namespace_de)
+    zh_sr = court_composition_setup(c.ZH_Steuerrekurs, ZH_Steuerrekurs_test_header, namespace_de)
+    zh_br = court_composition_setup(c.ZH_Baurekurs, ZH_Baurekurs_test_header, namespace_de)
+    zh_og = court_composition_setup(c.ZH_Obergericht, ZH_Obergericht_test_header, namespace_de)
+    zh_vg = court_composition_setup(c.ZH_Verwaltungsgericht, ZH_Verwaltungsgericht_test_header, namespace_de)
+    zh_svg = court_composition_setup(c.ZH_Sozialversicherungsgericht, ZH_Sozialversicherungsgericht_test_header, namespace_de)
+
+    test_data = [
+        # ZG_Verwaltungsgericht
+        (zg_vg.president.name, 'Adrian Willimann'),
+        (zg_vg.judges[0].name, 'Adrian Willimann'), 
+        # (zg_vg.judges[1].name, 'Jacqueline Iten-Staub'),
+        # (zg_vg.judges[2].name, 'Matthias Suter'),
+        # (zg_vg.clerks[0].name, 'Patrick Trütsch'),
```

Comment:
Are there reasons for these commented lines?

Reply:
Yes, these lines caused some index out of bounds errors during testing, so I commented them to see if the other tests run (which they do). The tests should all work after merging due to the changes I made in other PRs; and I meant to uncomment them again which I now did. 

Or should I have first merged those changes into this branch? I thought it was simpler not to do so if this branch has no conflicts with the base branch.

Reply:
I see. Makes sense, it's fine that way.